jpype/_jboxed.py

```diff
@@ -45,12 +47,21 @@ def __new__(cls, *args):
     __eq__ = int.__eq__
     __ne__ = int.__ne__
 
+    if _sys.version_info < (3,):
+        __cmp__ = int.__cmp__
```

Comment:
could you please add a comment here, why you do not use cmp in python3?

Reply:
I removed it from Python 3 because it is not being used.   At least according to the guides I could find that special method does nothing in Python 3.  I had it in both versions but I got a bug that showed up only in Python 2 and then did some research to discover the reason the bug didn't trigger in Python 3 was that nothing was calling it.  

https://portingguide.readthedocs.io/en/latest/comparisons.html

I don't really have an issue placing the `__cmp__` method in Python 3, but there is no guarantee that `int` will have a `__cmp__` at points in the future. 

Do you think that a comment to this effect is required in the code?